db/migrate/20170207174141_add_bet_option_to_user_bets.rb

```diff
@@ -0,0 +1,7 @@
+class AddBetOptionToUserBets < ActiveRecord::Migration[5.0]
+  def change
+    add_column      :user_bets, :bet_option, :integer
```

Comment:
Remember what we talked about yesterday, and make sure that the core you run actually does what you expect. 

This could also have been done in one step: http://apidock.com/rails/v4.0.2/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference 

Theres nothing wrong with doing it in multiple steps however I dont think this give you what you actually want